Cython/Debugger/Cygdb.py

```diff
@@ -104,67 +104,62 @@ def make_command_file(path_to_debug_info, prefix_code='',
 
     return tempfilename
 
-usage = "Usage: cygdb [options] [PATH [-- GDB_ARGUMENTS]]"
 
-def main(path_to_debug_info=None, gdb_argv=None, no_import=False):
+def main():
```

Comment:
I am for this change since this is `main()` function but not sure whether it won't break somewhere outside cython. Maybe I would go with it but I would like to see someone else opinion...

Reply:
Moreover, the arguments are changed. Again, I like new version but it can break users so I think another opinion is needed.